extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -212,6 +212,35 @@
   cursor: pointer !important;
 }
 
+.i-amphtml-lbg-thumbnail-timestamp-container {
+  background-color: #292D33;
+  color: #ffffff;
+  position: absolute;
+  bottom: 10px;
+  left: 10px;
+  height: 20px;
+  border-radius: 2px;
+  opacity: 0.8;
+  width: 20px;
+}
+
+.i-amphtml-lbg-thumbnail-timestamp-container.i-amphtml-lbg-has-timestamp {
+  font-size: 12px;
+  padding: 0 5px 0 18px;
+  line-height: 1.8;
+  width: auto;
+}
+
+.i-amphtml-lbg-thumbnail-play-icon {
+  mask-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor"><path d="M8 5v14l11-7z"/><path d="M0 0h24v24H0z" fill="none"/></svg>');
```

Comment:
What will this component look like on Edge where mask image is not supported?

Reply:
Yeah good point. I think the original intention here for `mask-image` was to be able to easily change the color of the svgs using `currentColor`. I'm not sure that's still possible with `background-image`, but since we don't have CSS API currently, I think that's ok. Maybe in the future we will use `mask-image` with a fallback to `background-image` for Edge.